packages/vuetify/src/directives/ripple/index.ts

```diff
@@ -204,7 +204,7 @@ function rippleHide (e: Event) {
     element._ripple.showTimerCommit = null
 
     // re-queue ripple hiding
-    element._ripple.showTimer = setTimeout(() => {
+    element._ripple.showTimer = window.setTimeout(() => {
```

Comment:
Please do not make unrelated changes to other files.

Reply:
I updated this line because it was throwing an error (Type 'Timeout' is not assignable to type 'number'.ts(2322)
) due to which I was unable to push my changes.